packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -577,8 +591,15 @@ async function processSQSMessageRequest(message) {
         if (!user) {
             throw new Error(`user not found: ${requestData.userId}`);
         }
+    } catch (err) {
+        log.error({ err }, 'Audit report generated by an invalid user');
+        return false;
+    }
+
+    try {
         await generateAndSendEmail(ARPA_REPORTER_BASE_URL, user.email, user.tenant_id);
     } catch (err) {
+        await sendErrorEmail(user)
         log.error({ err }, 'failed to generate and send audit report');
```

Comment:
Minor nit, but I like to throw logging as the first statement. Emails can sometimes have weird errors, so I like to get the log message out before something external happens.

Reply:
good call will update 👍